python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -14,10 +14,10 @@ def start_ganga():
     file_path = os.path.dirname(os.path.realpath(__file__))
     ganga_python_dir = os.path.join(file_path, '..', '..', '..')
     ganga_python_dir = os.path.realpath(ganga_python_dir)
-    sys.path.insert(0, ganga_python_dir)
+    if len(sys.path) >= 1 and ganga_python_dir != sys.path[0]:
+        sys.path.insert(0, ganga_python_dir)
```

Comment:
For my own interest, why is `sys.path.insert(0, ganga_python_dir)` not sufficient? Is it just to avoid adding it multiple times? In that case maybe:
```python
if ganga_python_dir not in sys.path:
    sys.path.insert(0, ganga_python_dir)
```
or do we really need our path to be at the beginning of the Python part?